backend/main.go

```diff
@@ -232,9 +233,12 @@ func main() {
 	highlight.SetProjectID("1jdkoe52")
 	if !util.IsOnPrem() && util.IsDevOrTestEnv() {
 		log.WithContext(ctx).Info("overwriting highlight-go graphql / otlp client address...")
-		highlight.SetOTLPEndpoint("http://localhost:4318")
-		if util.IsBackendInDocker() {
+		if otlpEndpoint != "" {
+			highlight.SetOTLPEndpoint(otlpEndpoint)
+		} else if util.IsBackendInDocker() {
 			highlight.SetOTLPEndpoint("http://collector:4318")
```

Comment:
We could even get rid of this part. But I didn't want to disrupt pre-existing workflows.

Reply:
nice! yeah, technically the env var should always be set now in docker to the same thing `"http://collector:4318"`, but it's fine to keep this to be safe in case we're in docker and `OTLP_ENDPOINT` is somehow not set